### PR TITLE
USWDS - Breadcrumb: Remove ability to set custom breadcrumb icons.

### PR DESCRIPTION
**Pull request by mejiaj**
## Description

Effort to provide good defaults for components. Removing the ability to set a custom separator and back icon (on mobile) is part of that work. Users can still override if they **really** want, but defining these defaults gives a universally recognized breadcrumb for users.

Uses utilities for placing icons introduced in #3531.

Takes care of the these tasks from #3413:
- [x] Remove settings from component settings file
- [x] Remove settings from component theme file
- [x] Update Breadcrumb Sass

### Removed settings
| Setting      | Description |
| ----------- | ----------- |
| $theme-breadcrumb-back-icon  | Set path to back arrow. |
| $theme-breadcrumb-back-icon-aspect   | Set the sizing for the back icon. |
| $theme-breadcrumb-icon-color | Set color for icons. |
| $theme-breadcrumb-separator-icon | Path to the separator icon. |
| $theme-breadcrumb-separator-icon-aspect | Aspect ratio to set sizing for separator icon. |

### Current settings
| Setting      | Description |
| ----------- | ----------- |
| $theme-breadcrumb-font-size | The size of the breadcrumb text. |
| $theme-breadcrumb-font-family | The family of the breadcrumb text. |
| $theme-breadcrumb-gap | The space between indicator icons and text. |
| $theme-breadcrumb-link-colors | The color of breadcrumb links in the form `[link], [hover], [active].` |
| $theme-breadcrumb-min-width | The width at which the breadcrumb displays the full breadcrumb list. |
| $theme-breadcrumb-spacing-bottom | Spacing below the breadcrumb. |
| $theme-breadcrumb-spacing-top | Spacing above the breadcrumb. |
| $theme-breadcrumb-wrapping | Controls whether the breadcrumb wraps or truncates. |

Before you hit Submit, make sure you’ve done whichever of these applies to you:

- [x] Follow the [18F Front End Coding Style Guide](https://pages.18f.gov/frontend/) and [Accessibility Guide](https://pages.18f.gov/accessibility/checklist/).
- [x] Run `npm test` and make sure the tests for the files you have changed have passed.
- [ ] Run your code through [HTML_CodeSniffer](http://squizlabs.github.io/HTML_CodeSniffer/) and make sure it’s error free.
- [x] Title your pull request using this format: [Website] - [UI component]: Brief statement describing what this pull request solves.

### CODE REVIEW

**Comment by mejiaj** [comment]
@thisisdano don't we need a `usa-breadcrumb--wrap` example in Fractal now that it's gone from a setting to a variation?

**Comment by thisisdano** [comment]
### Settings
| Setting      | Description |
| ----------- | ----------- |
| **$theme-breadcrumb-background-color** | The background color of the breadcrumb. (Necessary to set proper colors.)
| $theme-breadcrumb-font-size | The size of the breadcrumb text. |
| $theme-breadcrumb-font-family | The family of the breadcrumb text. |
| $theme-breadcrumb-gap | The space between indicator icons and text. |
| **$theme-breadcrumb-link-color** | The link color of breadcrumb links. (System will calculate hover, etc.) |
| $theme-breadcrumb-min-width | The width at which the breadcrumb displays the full breadcrumb list. |
| **$theme-breadcrumb-separator-color** | The color of the separator icons. |
| $theme-breadcrumb-spacing-bottom | Spacing below the breadcrumb. |
| $theme-breadcrumb-spacing-top | Spacing above the breadcrumb. |
| $theme-breadcrumb-wrapping | Controls whether the breadcrumb wraps or truncates. |
| **$theme-breadcrumb-spacing-x** |Spacing on the left and right of the breadcrumb.

### Variant
| Variant      | Description |
| ----------- | ----------- |
| **usa-breadcrumb--wrap** | Wrap the breadcrumb instead of truncating.

File: src/stylesheets/core/mixins/_icon.scss [hunk 1]
```diff
@@ -108,7 +116,7 @@
     map-get($icon-object, "path"),
     $theme-image-path
   );
-  $ie11-variant: get-token-from-bg($background-color, $color-alt, "black");
+  $ie11-variant: get-token-from-bg($contrast-bg, $color-variant, "black");
```
Comment (mejiaj) [inline src/stylesheets/core/mixins/_icon.scss:119]: Shouldn't this be`$contrast-bg → $contrast-background` based on `background-color: color($theme-tooltip-background-color);` from the tooltip updates?

File: src/stylesheets/core/mixins/_set-link-from-bg.scss [hunk 2]
```diff
@@ -13,8 +13,16 @@
 
   $link-token: nth($link-tokens, 1);
   $hover-token: nth($link-tokens, 2);
-
+  $visited-token: if(
+    abs(get-color-grade($theme-link-visited-color) - get-color-grade($bg-color))
```
Comment (mejiaj) [inline src/stylesheets/core/mixins/_set-link-from-bg.scss:17]: Similar to above. Shouldn't `$bg-` be spelled out?

**Comment by mejiaj** [comment]
There's a very tiny amount of clipping going on in IE11.

image

Possibly from
>IE11 is reported to not support calc() correctly in generated content
>IE11 is reported to have trouble with calc() with nested expressions, e.g. width: calc((100% - 10px) / 3); (i.e. it rounds differently)

Listed in known issues on https://caniuse.com/#feat=calc

File: src/stylesheets/core/mixins/_icon.scss (hunk 1 above)
Comment (thisisdano) [reply src/stylesheets/core/mixins/_icon.scss:119]: I like to spell out `background` in theme settings, but I don't think we have to in mixins, functions, and internal vars. We could do a thorough pass on `bg` → `background` perhaps in another (later) PR.

**Comment by thisisdano** [comment]
Hm — I'm inclined to let the `calc()` issue go _for now_ in IE11 — but I think we could try and address it in a point release.

**Comment by thisisdano** [comment]
Good catch, tho

**Comment by mejiaj** [comment]
Sounds good. I know it's a small issue, but just wanted to have it on record.

**Pull request merged**